doc-content/enterprise-only/websphere/kie-server-was-verify-proc.adoc

```diff
@@ -15,8 +15,8 @@ After you install {KIE_SERVER} and define the {KIE_SERVER} group mapping, verify
 +
 In these examples, replace the following placeholders:
 +
-** `<HOST>` is the ID or name of the {KIE_SERVER} host, for example, `localhost` or `192.7.8.9`.
+** `<HOST>` is the ID or name of the headless {PRODUCT_SHORT} controller, for example, `localhost` or `192.7.8.9`.
 
-**  `<PORT>` is the port number of the {KIE_SERVER} host, for example, `9060`.
+**  `<PORT>` is the port number of the headless {PRODUCT_SHORT} controller, for example, `9060`.
```

Comment:
This file was OK, this is about KIE Server installation.

Reply:
Thanks @MarianMacik . I changed it back.